astroquery/esa/hubble/tests/test_esa_hubble.py

```diff
@@ -93,9 +94,10 @@ def test_download_product_errors(self):
         assert "This product_type is not allowed" in err.value.args[0]
 
     def test_download_product_by_calibration(self):
+        tempdir = tempfile.mkdtemp("temp")
         parameters = {'observation_id': "J6FL25S4Q",
                       'cal_level': "RAW",
-                      'filename': "J6FL25S4Q.vot.test",
+                      'filename': tempdir + "/" + "J6FL25S4Q.vot.test",
```

Comment:
Does this work on Windows as intended or will `/` be interpreted as a part of the file name?

Reply:
Usually windows will automatically correct to a backslash if a forward slash is detected but sometimes this is not the case.

Reply:
I'll directly add a commit that changes this to `os.path.join`, and will go ahead and merge it then rather than go into more rounds.
Thanks @javier-ballester!